docs/installation_configuration_guide/model/Lite.md

```diff
@@ -8,13 +8,14 @@ sidebar_label: "Lite"
-| pollDuration          | Low        | duration | 100ms         | [Poll duration](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#poll(java.time.Duration)) of Kafka consumer | 
-| shutdownTimeout       | Low        | duration | 10s           | How long to wait for graceful shutdown                                                                                                               |
-| interpreterTimeout    | Low        | duration | 10s           | Timeout of invocation of scenario (including enrichers) for events consumed in one poll                                                              |
-| publishTimeout        | Low        | duration | 10s           | Timeout on producing resulting event to Kafka                                                                                                        |
-| waitAfterFailureDelay | Low        | duration | 10s           | Processing delay after unexpected, transient error (does not include e.g. expression errors or 500 codes from OpenAPI)                               |
+| Name                     | Importance | Type     | Default value | Description                                                                                                                                         |
+|--------------------------|-----------|----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| pollDuration             | Low       | duration | 100ms         | [Poll duration](https://kafka.apache.org/30/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#poll(java.time.Duration)) of Kafka consumer | 
+| shutdownTimeout          | Low       | duration | 10s           | How long to wait for graceful shutdown                                                                                                              |
+| interpreterTimeout       | Low       | duration | 10s           | Timeout of invocation of scenario (including enrichers) for events consumed in one poll                                                             |
+| publishTimeout           | Low       | duration | 10s           | Timeout on producing resulting event to Kafka                                                                                                       |
+| waitAfterFailureDelay    | Low       | duration | 10s           | Processing delay after unexpected, transient error (does not include e.g. expression errors or 500 codes from OpenAPI)                              |
+| kafkaTransactionsEnabled | High      | boolean  | true          | Define if records should be processed in transaction (exactly-once semantics) or not (at-least-once semantics)                                      |
```

Comment:
also, should we write here that if Azure Kafka is detected default is false?

Reply:
Done